src/collections/_documentation/apm/apm-glossary.md

```diff
@@ -0,0 +1,170 @@
+---
+title: APM Glossary
+sidebar_order: 5
+---
+
+### Hierarchy
+
+Each span can be connected to other spans using a parent-child hierarchy. 
+
+For example, a span called `updateJiraTask` would have child spans like `Check permissions`, `Update Task`, `sql.update`, `send.webhooks`.
+
+Each sub-task would have an id in the attribute `parent_span_id` id of the “updateTask” span.
+
+### Properties
+
+Each span has a unique identifier kept in the `span_id` attribute.
+
+Start time and end time are stored in the `start_timestamp` and `timestamp`. End time is captured in the attribute timestamp and not the end_timestamp for now because of compatibility reasons.
+
+The name of the operation is stored in the “op” parameter. Examples are `http`, `sql.query`, `redis.command`.
+
+Additional data about the operation can be stored in the `data` and `tags` attributes.
+
+### Span
+
+The **span** is the primary building block of a distributed trace, representing an individual unit of work done in a distributed system.
+
+It can be the processing of an HTTP request by a server, the querying of a database, or an HTTP call to an external service.
+
+An example of a span that describes a call to an external service:
+
+    {
+          "trace_id": "a0fa8803753e40fd8124b21eeb2986b5",
+          "parent_span_id": "9c2a6db8c79068a2",
+          "span_id": "8c931f4740435fb8",
+          "start_timestamp": 1563890702.134,
+          "same_process_as_parent": true,
+          "description": "http://httpbin.org/base64/aGVsbG8gd29ybGQK GET",
+          "tags": { "http.status_code": 200, "error": false },
+          "timestamp": "1563890702.934",
```

Comment:
Where does this example come from? I do not understand why the timestamp is both a unix timestamp and sent as a string. This seems to be invalid data as per the schema

Reply:
I got it from the forum docs. Navigate to the bottom Glossary and decompress the toggle: https://forum.sentry.io/t/sentrys-apm-docs-alpha/7843

Reply:
can we still drop the quotes around the number here? This is wrong and is unlikely to be accepted by the server. Fairly certain none of our SDKs send this

Reply:
nvm this is tracked as part of a different convo